xpk/README.md

```diff
@@ -106,6 +106,53 @@ cleanup with a `Cluster Delete`.
     --cluster xpk-test
     ```
 
+# How to add docker images to a xpk workload
+
+The default behavior is that the local running directory will be built into
+a docker image by xpk and those files are accessable by `xpk workload create`.
+See below for the two ways that xpk supports configuring docker images for the
+workload. Do not mix arguments from these two ways like `--base-docker-image`
+and `--docker-image` in the same command.
+
+## Recommended / Default Docker Flow: `--base-docker-image` and `--script-dir`
+Pull in a local directory with scripts or files interested in running in
+`xpk workload create`.
+
+* `--base-docker-image` to set the base image that xpk will start with.
+* `--script-dir` to set which directory to pull into the image.
+* See `python3 xpk/xpk.py workload create --help` for more.
+
+Example with defaults which pulled the local directory into the base image:
```

Comment:
Can we add a second one that has --base-docker-image=gcr.io/your_dependencies_docker_image and say that is the recommended flow for <10k accelerators?

Reply:
Done!